main/SQLHelper.cpp

```diff
@@ -3052,6 +3052,14 @@ bool CSQLHelper::OpenDatabase()
 					_log.Log(LOG_STATUS, "A default admin User called 'admin' has been added with a default password!");
 				}
 			}
+			// Remove these Pref vars as we do not use them anymore
+			DeletePreferencesVar("EnableTabLights");
```

Comment:
Could you increase the DB_VERSION and make this a patch? (see examples like if (debversion < 155) etc